chrooted-iso.sh

```diff
@@ -1,25 +1,39 @@
 #!/usr/bin/env bash
 echo "Chrooted in the new system, running as $(whoami)"
 
-# Compile GSchemas
-glib-compile-schemas /usr/share/glib-2.0/schemas
-
 # User setup
 useradd -mG wheel crystal
 usermod -c "Password // \"crystal\"" crystal
 usermod -p $(echo "crystal" | openssl passwd -6 -stdin) crystal
 usermod -p $(echo "crystal" | openssl passwd -6 -stdin) root
 chsh -s /usr/bin/zsh crystal
 
-# Install Jade's GUI
+# Install Jade GUI
 flatpak install -y --noninteractive /usr/share/jade-gui/jade-gui.flatpak
 
 # Desktop icon for Jade's GUI
 mkdir -p /home/crystal/Desktop
 cp \
   /var/lib/flatpak/exports/share/applications/al.getcryst.jadegui.desktop \
   /home/crystal/Desktop/Install.desktop
+
+# Disable auto screen lock
+mkdir -p /home/crystal/.config/autostart
+echo "[Desktop Entry]
+Name=Deactive lock screen
+Comment=Deactive the gnome lock screen in the live session
+Type=Application
+Icon=nautilus
+Exec=sh -c \"gsettings set org.gnome.desktop.screensaver lock-enabled false\"" > /home/crystal/.config/autostart/no-lock-screen.desktop
+
+# Jade-GUI Autostart
+cp \
+  /var/lib/flatpak/exports/share/applications/al.getcryst.jadegui.desktop \
+  /home/crystal/.config/autostart
+
+# Permissions for Crystal user
 chown -R crystal:crystal /home/crystal/
+chmod +x /home/crystal/.config/autostart/*.desktop
```

Comment:
.desktop files usually don't need the executable bit

Reply:
They actually require it for autostart

Reply:
oh interesting